### ISSUE TITLE
Only twig files

### ISSUE THREAD

**Issue by palmiak** [palmiak/timber-acf-wp-blocks#18]
I'm usually working in Drupal land, but recently getting into Wordpress and this repo/PR is making it so much easier!

I just had to switch the `DirectoryIterator` to skip over non-twig files in order to use the folder structure example in #10.

```php
$file_parts = pathinfo($template->getFilename());

if ($file_parts['extension'] !== 'twig') {
  continue;
}

// Strip the file extension to get the slug.
$slug = $file_parts['filename'];
```

Thanks so much!

_Originally posted by @BrentWMiller in https://github.com/palmiak/timber-acf-wp-blocks/pull/15#issuecomment-575438185_

**Comment by palmiak** [comment]
@BrentWMiller thanks - I'll try to add a fix for this soon.

**Comment by palmiak** [comment]
Closed by #15
